src/common/utils/chain-utils.ts

```diff
@@ -55,6 +55,39 @@ export const walletSwitchChain = async (chainId: ChainId): Promise<void> => {
       // Possibly on localhost which doesn't support the call
       return console.error(e);
     }
+    if (e.code === 4902) {
+      return walletAddChain(chainId);
+    }
+    throw e;
+  }
+};
+
+/**
+ * Adds network to user's wallet.
+ * Networks with no RPC URL provided will not be added, particularly the default chains that already comes with Metamask.
+ * @param chainId Chain ID of target network
+ */
+export const walletAddChain = async (chainId: ChainId): Promise<void> => {
```

Comment:
wanna test this?

Reply:
Added tests for `walletAddChain` in https://github.com/TradeTrust/tradetrust-website/pull/582/commits/f9a9c35f0c864df25ab5f66aa063c5ac79d6d9a8.

Reply:
sweet